client/guidestours/index.js

```diff
@@ -183,22 +290,32 @@ export default class GuidesTours extends Component {
 			return null;
 		}
 
-		const { bullseye, dialog } = this.getStepPositions();
-		const stepCoords = posToCss( dialog );
-		const pointerCoords = posToCss( bullseye );
+		let StepComponent = GuidesStep;
+		switch ( this.state.currentStep.type ) {
+			case 'GuidesFirstStep':
```

Comment:
I think it's worth exploring, but after a few minutes of poking it with a stick I also think it's worth deferring. WDYT, does this need to get in with this PR?